_posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md

```diff
@@ -26,6 +26,7 @@ The following **open-source servers** are available, if you want to run your own
 - [Node Solid Server](https://github.com/solid/node-solid-server) for Node.js
 - [solid-nextcloud](https://github.com/pdsinterop/solid-nextcloud) as a plugin for Nextcloud
 - [PHP Solid Server](https://github.com/pdsinterop/php-solid-server) for PHP
+- [Manas](https://github.com/manomayam/manas) for Rust (in alpha stage)
```

Comment:
I'd suggest in this case to leave out project's status information as that would change, and we don't need to maintain that in this document. People can always find the most current state of the project at the source.

```suggestion
- [Manas](https://github.com/manomayam/manas) for Rust
```